applets/notification_area/main.c

```diff
@@ -259,6 +352,9 @@ na_tray_applet_dispose (GObject *object)
 #endif
 
   G_OBJECT_CLASS (na_tray_applet_parent_class)->dispose (object);
+
+  if (&NA_TRAY_APPLET (object)->priv->builder)
+    g_clear_object (&NA_TRAY_APPLET (object)->priv->builder);
```

Comment:
`g_clear_object` is `NULL` safe so you don't need to check if builder is NULL before calling g_clear_object.

Reply:
:+1: